tests/test_on_data/test_imaging_interfaces.py

```diff
@@ -54,3 +59,82 @@ class TestSbxImagingInterface(ImagingExtractorInterfaceTestMixin, TestCase):
         dict(file_path=str(OPHYS_DATA_PATH / "imaging_datasets" / "Scanbox" / f"sample.sbx")),
     ]
     save_directory = OUTPUT_PATH
+
+
+class TestBrukerTiffImagingInterface(ImagingExtractorInterfaceTestMixin, TestCase):
+    data_interface_cls = BrukerTiffImagingInterface
+    interface_kwargs = dict(
+        folder_path=str(
+            OPHYS_DATA_PATH / "imaging_datasets" / "BrukerTif" / "NCCR32_2023_02_20_Into_the_void_t_series_baseline-000"
+        )
+    )
+    save_directory = OUTPUT_PATH
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.device_metadata = dict(name="BrukerFluorescenceMicroscope", description="Version 5.6.64.400")
+        cls.optical_channel_metadata = dict(
+            name="Ch2",
+            emission_lambda=np.NAN,
+            description="An optical channel of the microscope.",
+        )
+        cls.imaging_plane_metadata = dict(
+            name="ImagingPlane",
+            description="The plane imaged at 5e-06 meters depth.",
+            excitation_lambda=np.NAN,
+            indicator="unknown",
+            location="unknown",
+            device=cls.device_metadata["name"],
+            optical_channel=[cls.optical_channel_metadata],
+            imaging_rate=30.345939461428763,
+            grid_spacing=[1.1078125e-06, 1.1078125e-06],
+        )
+
+        cls.two_photon_series_metadata = dict(
+            name="TwoPhotonSeries",
+            description="Imaging data acquired from the Bruker Two-Photon Microscope.",
+            unit="px",
+            dimension=[512, 512],
+            imaging_plane=cls.imaging_plane_metadata["name"],
+            format="tiff",
+            scan_line_rate=15840.580398865815,
+            field_of_view=[0.0005672, 0.0005672, 5e-06],
+        )
+
+        cls.ophys_metadata = dict(
+            Device=[cls.device_metadata],
+            ImagingPlane=[cls.imaging_plane_metadata],
+            TwoPhotonSeries=[cls.two_photon_series_metadata],
+        )
+
+    def check_extracted_metadata(self, metadata: dict):
+        self.assertEqual(metadata["NWBFile"]["session_start_time"], datetime(2023, 2, 20, 15, 58, 25))
+        self.assertEqual(metadata["Ophys"], self.ophys_metadata)
```

Comment:
Does `assertEqual` on two `dict` objects automatically call `assertDictEqual`? Or is it literally just asserting `==`? If so please use the most specific test possible, such as `assertDictEqual` or `assertCountEqual`

Reply:
`assertEqual` will automatically call `assertDictEqual`. `assert dict1 == dict2` will also work. See [this blog post](https://kracekumar.com/post/five-reason-to-use-pytest/), which makes a pretty strong case for just using pytest for these sorts of things